pmm-tests/pmm-framework.sh

```diff
@@ -706,6 +730,25 @@ setup(){
   echo -e "******************************************************************"
 }
 
+#Download and install PMM2.x client
+install_client(){
+ PMM_CLIENT_TAR_URL=$(lynx --listonly --dump https://www.percona.com/downloads/TESTING/pmm/ | grep  "pmm2-client" |awk '{print $2}'| grep "tar.gz" | head -n1)
+  echo "PMM2 client  $PMM_CLIENT_TAR_URL"
+  wget $PMM_CLIENT_TAR_URL
+  PMM_CLIENT_TAR=$(echo $PMM_CLIENT_TAR_URL | grep -o '[^/]*$')
+  tar -xzf $PMM_CLIENT_TAR
+  PMM_CLIENT_BASEDIR=$(ls -1td pmm2-client-* 2>/dev/null | grep -v ".tar" | head -n1)
+  pushd $PMM_CLIENT_BASEDIR > /dev/null
+  export PATH="$PWD/bin:$PATH"
+
```

Comment:
please fix indentation here and extra space

Reply:
extra space and indentation were fixed